### DISCUSSION TITLE
Please add ISIN Numbers

### DISCUSSION THREAD

**Discussion by Jleventhal1** [OpenBB-finance/OpenBB#5327] [Ideas]
Hello, please include securities' ISIN numbers to the SDK

**Comment by jmaslek** [comment]
So hopefully we will have a more consistent way of handling things like ISIN (it depends on provider to provider if they support it).

You can currently (at least for my quick testing on US stocks) use the sdk with ISIN numbers for stocks.load and if you want to use it through other functions, you can get the ticker from the isin by doing (where func is an openbb that takes a symbol)

```
from openbb_terminal.sdk import openbb
import yfinance
openbb.func(symbol = yf.Ticker( YOUR ISIN NUMBER ).ticker , **other kwargs) 
```
